org.jacoco.doc/docroot/doc/integrations.html

```diff
@@ -40,8 +40,8 @@
     </tr>
     <tr>
       <td>Command Line</td>
-      <td><a href="agent.html">JaCoCo Manual</a></td>
-      <td></td>
+      <td><a href="agent.html">JaCoCo Java agent</a>, <a href="cli.html">JaCoCo Command Line Interface</a></td>
+      <td>Since version 0.8.0</td>
```

Comment:
@marchof given that version is only about CLI, I think will be better to have separate entries for Agent and CLI